### PR TITLE
Added eye test expense information

### PR DESCRIPTION
**Pull request by richmadetech**
Added guidance around expensing your eye test and purchasing Lenses/Frames. 

Guidance taken from here: https://madetechteam.slack.com/archives/C8M1K094L/p1661156588475639?thread_ts=1661154750.279709&cid=C8M1K094L

@JillKittos / @Vikkigill  could you confirm this guidance is correct please :)

### CODE REVIEW

File: guides/compensation/expenses/eyetest.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Eye Test Expenses
+
+You can expense the cost of an annual Eye Test through Xero.
+
+- Book the eye test yourself and get a reciept
+- Submit the reciept for the eye test through Xero
```
Comment (cob16) [inline guides/compensation/expenses/eyetest.md:6]:
```suggestion
- Submit the receipt for the eye test through Xero
```

File: guides/compensation/expenses/eyetest.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Claiming for Lenses and Frames
+
+If the outcome of the eye test is that you require glasses for the use of Display Screen Equipment (DSE), then Made Tech will also make a contribution towards the costs of a pair of glasses.
+
+- Made Tech will pay for the lenses plus an additional £60 toward any frames that you choose.
+- After you purhcase your frames and lenses, submit the receipts through Xero.
```
Comment (cob16) [inline guides/compensation/expenses/eyetest.md:13]:
```suggestion
- After you purchase your frames and lenses, submit the receipts through Xero.
```

File: guides/compensation/expenses/eyetest.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Note: If you use Westfield Health you may want to reclaim the cost using their service instead, especially if you wish to purchase more expensive frames.
```
Comment (cob16) [inline guides/compensation/expenses/eyetest.md:15]:
```suggestion
**Note:** If you use Westfield Health you may want to reclaim the cost using their service instead, especially if you wish to purchase more expensive frames.
```

**Comment by JillKittos** [comment]
Employees can claim back the costs of an annual eyes test under Regulation 5 of the DSE rules that state 'employers must provide an eye exam and vision tests, on request, to all current or new DSE users'. If you book an appointment with your optician, you can claim the annual eye test back via expenses through Xero. Made Tech will pay for glasses for DSE work if the test shows an employee needs special glasses prescribed for the distance the screen is viewed at. Made Tech will contribute £60 towards a basic pair of corrective glasses. If an ordinary prescription is suitable, you cannot claim via Made Tech expenses.
Please request that the optician’s DSE eyesight test incorporates a check on the DSE distance, and if the optician feels that corrective glasses are required, ask that they provide a report/copy of the prescription indicating these are explicitly required for DSE use

**Pull request merged**